applications/StructuralMechanicsApplication/custom_constitutive/truss_constitutive_law.cpp

```diff
@@ -141,6 +141,7 @@ void TrussConstitutiveLaw::CalculateMaterialResponsePK2(Parameters& rValues)
     Vector& stress_vector = rValues.GetStressVector();
     if (stress_vector.size() != 1) stress_vector.resize(1, false);
     stress_vector[0] = this->CalculateStressElastic(rValues);
+    AddInitialStressVectorContribution(stress_vector);
```

Comment:
This change can be ignored as it will be redundant when the PR for structural is merged